client/preprocessors.py

```diff
@@ -317,17 +317,17 @@ def run(self):
             for section in toc['sections']:
                 markdown += self.compile_section(project, section, 2)
             markdown = self.fix_links(markdown)
-            output_file = os.path.join(self.output_dir, '{0}.md'.format(project.identifier))
+            output_file = os.path.join(self.output_dir, '{0}-{1}.md'.format(idx+1, project.identifier))
```

Comment:
Yeah, I thought about that too and was about to pad it in my original code. Will do.